src/main/scala/sbtecr/EcrPlugin.scala

```diff
@@ -7,6 +7,8 @@ import sbt._
 import scala.language.postfixOps
 
 object EcrPlugin extends AutoPlugin {
+  
+  lazy val version:SettingKey[String] = settingKey[String]("The ECR docker image version.")
```

Comment:
I am unsure of all this (I am still learning sbt), but I think a key must be in autoimport unless it is one of the [defaults](https://github.com/sbt/sbt/blob/517cbf6d2d7239032daf2b4d0b312aebc7662341/main/src/main/scala/sbt/Keys.scala) or is imported by another plugin.  Otherwise it does not exist in scope and cannot be set. My previous comment was incorrect.  version is a default key not a key defined by sbt-native.

This leads me to think that key names should either match a default key such as version, match a required key of sbt-native (since sbt-ecr requires sbt-native) or have unique names that are unlikely to conflict with other plugins.

I also realized that version should not be redefined in the plugin it should be imported from sbt.Keys.

I will update the pull request.